packages/libdisplay-info-git/libdisplay-info-git

```diff
@@ -0,0 +1,30 @@
+pkgname="libdisplay-info-git"
+gives="libdisplay-info-dev"
+provides=("libdisplay-info1" "libdisplay-info1")
+replaces=("libdisplay-info1" "libdisplay-info1" "libdisplay-info-dev")
+breaks="${replaces}"
+pkgver="0.1.1"
+pkgdesc='EDID and DisplayID library'
+url='https://gitlab.freedesktop.org/emersion/libdisplay-info'
+arch=("amd64")
+license=('MIT')
+source=("git+https://gitlab.freedesktop.org/emersion/libdisplay-info")
+sha256sums=('SKIP')
+makedepends=("git" "hwdata" "meson" "ninja-build" "python3")
+conflicts=(libdisplay-info)
+provides=(libdisplay-info libdisplay-info.so)
```

Comment:
Currently we don't support `.so` dependencies.

Reply:
Lines are duplicated....